misc/Tremulous-Grangerhub.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Tremulous
+Comment=Aliens vs Humans, First Person Shooter game with elements of Real Time Strategy
+Icon=io.github.grangerhub.Tremulous
```

Comment:
If you really want to use the "Tremulous-Grangerhub" naming scheme (which I strongly disagree with as I mentioned), you need to fix the Icon name here as well.